### PR TITLE
pify

### PR DESCRIPTION
**Pull request by SamVerschueren**

### CODE REVIEW

File: cli.js [hunk 1]
```diff
@@ -1,31 +1,29 @@
...
 
 if (!username) {
 	console.error('Please supply an npm username');
 	process.exit(1);
 }
 
-npmEmail(username, function (err, email) {
-	if (err) {
+npmEmail(username)
+	.then(email => {
```
Comment (sindresorhus) [inline cli.js]:
``` js
.then(console.log)
```

File: cli.js [hunk 2]
```diff
@@ -1,31 +1,29 @@
...
+		console.log(email);
+	})
+	.catch(err => {
 		console.error(err.message);
 		process.exit(1);
```
Comment (sindresorhus) [inline cli.js]: Meow does this for us, so we don't need.

File: readme.md [hunk 3]
```diff
@@ -13,15 +13,28 @@ $ npm install --save npm-email
...
 	//=> 'sindresorhus@gmail.com'
 });
 ```
 
 
+## API
+
+### npmEmail(username)
+
+Returns a promise for the user's email adres.
```
Comment (sindresorhus) [inline readme.md]: address

**Comment by SamVerschueren** [comment]
fixed

File: .travis.yml [hunk 4]
```diff
@@ -1,6 +1,6 @@
 sudo: false
 language: node_js
 node_js:
-  - 'iojs'
-  - '0.12'
-  - '0.10'
+  - '6'
+  - '5'
```
Comment (SamVerschueren) [inline .travis.yml]: drop `5`?

Comment (sindresorhus) [inline .travis.yml]: Yup

Comment (SamVerschueren) [inline .travis.yml]: done 👍

**Comment by sindresorhus** [comment]
🎉

**Comment by SamVerschueren** [comment]
🍹

**Pull request merged**